data/es/maps/level_0/map_properties.tmx

```diff
@@ -3,6 +3,6 @@
         <property name="chapter_id" type="int" value="1"/>
         <property name="level_name" value="Vandalismo en la Cripta"/>
         <property name="main_mission_description" value="Salir de la Aldea"/>
-        <property name="main_mission_type" value="POSITION"/>
+        <property name="main_mission_type" value="POSICIÓN"/>
```

Comment:
I don't think it's something that should be translated.

It's a constant used in code, not a value displayed to the player as far as I know.